framework/examples/transport/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
+find_package(elpp REQUIRED)
```

Comment:
It used to work but probably got broken long time ago :)

Reply:
Indeed, I tried on the master branch and it also breaks in the same way.